src/Codeception/Module/Db.php

```diff
@@ -157,26 +178,30 @@ protected function removeInserted()
     {
         foreach ($this->insertedIds as $insertId) {
             try {
-            $this->driver->deleteQuery($insertId['table'], $insertId['id']);
+                $pdo  = $this->driver->getDbh();
+                $stmt = $pdo->prepare(
+                  'DELETE FROM `' . $insertId['table'] . '` WHERE `' . $insertId['primary'] . '` = ?'
```

Comment:
Is there a reason why you use raw query here? I think it would be better to move SQL outside Db module into db drivers we have.